internal/planner/planner_test.go

```diff
@@ -74,6 +75,77 @@ func TestPlannerHelloWorld(t *testing.T) {
 			note:    "sets",
 			queries: []string{"x = {1,2,3}; x[y]"},
 		},
+		{
+			note:    "vars",
+			queries: []string{"x = 1"},
+		},
+		{
+			note:    "complete rules",
+			queries: []string{"true"},
+			modules: []string{`
+				package test
+				p = x { x = 1 }
+				p = y { y = 2 }
+			`},
+		},
+		{
+			note:    "complete rule reference",
+			queries: []string{"data.test.p = 10"},
+			modules: []string{`
+				package test
+				p = x { x = 10 }
+			`},
+		},
+		{
+			note:    "functions",
+			queries: []string{"data.test.f([1,x])"},
+			modules: []string{`
+				package test
+				f([a, b]) {
+					a = b
+				}
+			`},
+		},
+		{
+			note:    "else",
+			queries: []string{"data.test.p = 1"},
+			modules: []string{`
+				package test
+				p = 0 {
+					false
+				} else = 1 {
+					true
+				}
+			`},
+		},
+		{
+			note:    "partial set",
+			queries: []string{"data.test.p = {1,2}"},
+			modules: []string{`
+				package test
+				p[1]
+				p[2]
+			`},
+		},
+		{
+			note:    "partial object",
+			queries: []string{`data.test.p = {"a": 1, "b": 2}`},
+			modules: []string{`
+				package test
+				p["a"] = 1
+				p["b"] = 2
+			`},
+		},
+		{
+			note:    "virtual extent",
+			queries: []string{`data`},
+			modules: []string{`
+				package test
+
+				p = 1
+			#	q = 2 { false }
```

Comment:
nit: was this supposed to be commented out?

Reply:
Fixed this in a follow-up PR. This was mostly for test purposes. Fixed now.